### PR TITLE
Usage improvements

### PR DESCRIPTION
**Pull request by mrozlukasz**
Changed the usage for command line gradle task that accepts java jars to scan. Example:
```
gradle execute -Dargs='--archive=/some/dir/to/archive.jar --package=io.github.jhipster.jpa2jdl.example.entities'

gradle execute -Dargs='--archive=/some/dir/to/archive.jar --archiveRoot=/BOOT-INF/classes/ --package=io.github.jhipster.jpa2jdl.example.entities'
```

### CODE REVIEW

File: src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    private String archiveFile;
+
+    @Option(name="--archiveRoot", required = false)
+    private String archiveRoot = "/";
+
+    @Option(name="--out", required = true)
+    private String outputpath;
+
+    public static void main(String[] args) {
+        Arrays.stream(args).forEach(System.out::println);
```
Comment (yveshwang) [inline src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java]: try to avoid unnecessary print outs to better support piping and redirection.

File: src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+import java.net.URLClassLoader;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+
+public class NewRunner {
```
Comment (yveshwang) [inline src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java]: whats the rationale for a new class called "NewRunner" instead of the old main class? renaming classes and creating unnecessary separation of concerns IMO makes code changeset hard to follow. i understand everybody has their own fav structure/pattern they wanna impose, but lets try to follow the original authors structure if it is simply naming and stylistic change.

File: src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    @Option(name="--package", required = true)
+    private String rootPackage;
+
+    @Option(name="--archive", required = true)
+    private String archiveFile;
+
+    @Option(name="--archiveRoot", required = false)
+    private String archiveRoot = "/";
+
+    @Option(name="--out", required = true)
```
Comment (yveshwang) [inline src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java]: unrequired.

File: src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+    private ReverseJPA2JDL jpa2jdl = new ReverseJPA2JDL();
+
+    @Option(name="--package", required = true)
+    private String rootPackage;
+
+    @Option(name="--archive", required = true)
+    private String archiveFile;
+
+    @Option(name="--archiveRoot", required = false)
```
Comment (yveshwang) [inline src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java]: and new param options all need default value and description please.

Comment (mrozlukasz) [reply src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java]: This should be considered as a tool, rather than software to generate jdl from itself, hence I think there is no point to use it like it was before. What default value should be used for pointing jar archive? I cannot imagine default value here.

File: src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java (hunk 1 above)
Comment (mrozlukasz) [reply src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java]: :+1:

File: src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java (hunk 2 above)
Comment (mrozlukasz) [reply src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java]: This was my leftover that I created a new class side by side to the old one. Renamed
